### PR TITLE
docs: added sample gorillamux-redis app

### PR DESCRIPTION
**Pull request by developer-diganta**
# Pull Request Template

## Description

Added sample gorilla-mux redis sample app.

## Additional Context (Please include any Screenshots/gifs if relevant)

image
## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.

### CODE REVIEW

File: docs/go/quickstart/gorillamux-redis.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+redis-server
+```
+
+### Start Keploy Record Mode and run the application
+
+```
+export KEPLOY_MODE=record && go run main.go
+```
+
+### Skip above steps with Gitpod
```
Comment (nehagup) [inline docs/go/quickstart/gorillamux-redis.md:44]: Is it for gorilla-mux app?

File: docs/go/quickstart/gorillamux-redis.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+```
+
+> **We got 71.4% without writing any e2e testcases or mocks for Redis!**
+
+So no need to setup fake database/apis like Redis or write mocks for them. Keploy automatically mocks them and, **The application thinks it's talking to Redis 😄**
+
+Go to the `Keploy Console` to get deeper insights on what testcases ran, what failed.
+
+<details>
+<summary>𝗜𝗻𝘀𝗶𝗴𝗵𝘁𝘀 𝗼𝗻 𝗞𝗲𝗽𝗹𝗼𝘆 𝗖𝗼𝗻𝘀𝗼𝗹𝗲</summary>
```
Comment (nehagup) [inline docs/go/quickstart/gorillamux-redis.md:175]: can you also add an image?

File: docs/go/quickstart/gorillamux-redis.md [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+<details>
+<summary>𝗞𝗲𝗽𝗹𝗼𝘆 𝗟𝗼𝗴𝘀</summary>
+
+```shell
+ <=========================================>
+  TESTRUN STARTED with id: "5ae0c256-f54d-4126-a794-5d5f50d3db76"
+	For App: "my-app"
+	Total tests: 2
+ <=========================================>
+
```
Comment (nehagup) [inline docs/go/quickstart/gorillamux-redis.md:239]: Please add image

File: docs/go/quickstart/gorillamux-redis.md (hunk 1 above)
Comment (developer-diganta) [reply docs/go/quickstart/gorillamux-redis.md:44]: Yes

**Comment by developer-diganta** [comment]
@nehagup can you please review? I have made the changes.

**Review by nehagup — APPROVED** [review]

**Pull request merged**